backend/endpoints/responses/platform.py

```diff
@@ -15,5 +17,8 @@ class PlatformSchema(BaseModel):
     logo_path: str | None = ""
     firmware: list[FirmwareSchema] = Field(default_factory=list)
 
+    created_at: datetime
+    updated_at: datetime
```

Comment:
come to think of it, these could all go in BaseModel, and every table would have created and updated columns

Reply:
good idea, i'll add it to the base model!